Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -274,45 +333,51 @@ same way like the other examples:
 Include individual extensions like site packages
 ================================================
 
-.. todo:: This is not allowed anymore. The sitepackage has to go to another
-   directory and be included via composer properly
+A project will often contain custom extensions, and at the least a :ref:`sitepackage <t3sitepackage:start>`
+that provides the TYPO3-related project templates and configuration.
 
-It's not necessary to move your project's site package to a dedicated
-Git repository to re-include it in your project. You can keep the files
-in your main project (e.g. :file:`public/typo3conf/ext/my_sitepackage`). There is
-only one thing to do; Because TYPO3's autoload feature works differently
-in Composer based installations, you have to register your PHP class
-names in Composer. This is very easy when you use PHP namespaces:
+Previously, this was stored in a directory like :file:`typo3conf/ext/my_sitepackage`.
+In Composer mode, you can easily add a custom repository within your project
+of the type `path`, so that you can require your sitepackage as if it was
+a normal package. You would not need to put it into a distinct Git or remote
+Composer repository.
```

Comment:
What is a "remote Composer repository"?

Reply:
Will rephrase to "Packagist or other custom remote repository like GitHub".

This should differentiate that you dom't need to create any remote repository, but a local one.